R/make-array.R

```diff
@@ -15,6 +15,10 @@
 #' @param selections character, for `makeMR` and `deriveArray` the names of the
 #' categories to mark as the dichotomous selections. Required for
 #' `makeMR`; optional for `deriveArray`; ignored in `makeArray`.
+#' @param numeric Logical indicating whether the array should be a numeric
+#' array or categorical array. `NULL` the default will guess numeric if
+#' all variables are known to be numeric and categorical if all are
+#' categorical. It cannot guess for expression subvariables.
```

Comment:
`It cannot guess if the subvariables are expressions rather than true Crunch Variables` or maybe something better?